src/pages/booklist/components/Textbook.tsx

```diff
@@ -49,77 +39,82 @@ export function Textbook({
-            <Tr>
-              <Td p="0" w="10px">
-                <Text as="strong" mr="4">
-                  NEW & DIGITAL ACCESS
-                </Text>
-              </Td>
-              <Td p="0">{newAndDigitalAccessCad}</Td>
-            </Tr>
-          )}
-          {digitalAccessCad && (
-            <Tr>
-              <Td p="0" w="10px">
-                <Text as="strong" mr="4">
-                  DIGITAL ACCESS
-                </Text>
-              </Td>
-              <Td p="0">{digitalAccessCad}</Td>
-            </Tr>
-          )}
-          <Tr>
-            <Td pl="0" pt="10">
-              <Text as="strong">ISBN</Text>
-            </Td>
-            <Td pl="0" pt="10">
-              {isbn}
-            </Td>
-          </Tr>
-        </Table>
-      </VStack>
-      <Spacer />
-      <VStack ml="3" w="10em">
+          <HStack>
+            <Heading size="sm" maxW={{ base: '18rem', md: '100%' }} textAlign={{ base: 'center', md: 'left' }}>
+              {title}
+            </Heading>
+          </HStack>
+          <Text fontSize="sm" textAlign={{ base: 'center', md: 'left' }}>
+            {authors?.join(', ')}
+          </Text>
+          <Table w="fit-content" size="sm" variant="unstyled" mt="2.25rem!important">
+            <Tbody display="flex" flexDirection="column" gap="2">
+              {newCad && (
+                <Tr display="flex" gap="2">
+                  <Td p="0" w="5rem">
+                    <Text as="strong">NEW</Text>
+                  </Td>
+                  <Td p="0">{newCad}</Td>
+                </Tr>
+              )}
+              {usedCad && (
+                <Tr display="flex" gap="2">
+                  <Td p="0" w="5rem">
+                    <Text as="strong">USED</Text>
+                  </Td>
+                  <Td p="0">{usedCad}</Td>
+                </Tr>
+              )}
+              {newAndDigitalAccessCad && (
+                <Tr display="flex" gap="2">
+                  <Td p="0" w="5rem">
+                    <Text as="strong">NEW & DIGITAL ACCESS</Text>
+                  </Td>
+                  <Td p="0">{newAndDigitalAccessCad}</Td>
+                </Tr>
+              )}
+              {digitalAccessCad && (
+                <Tr display="flex" gap="2">
+                  <Td p="0" w="5rem">
+                    <Text as="strong">DIGITAL ACCESS</Text>
+                  </Td>
+                  <Td p="0">{digitalAccessCad}</Td>
+                </Tr>
+              )}
+              <Tr display="flex" gap="2">
+                <Td p="0" w="5rem">
+                  <Text as="strong">ISBN-13</Text>
+                </Td>
+                <Td p="0">{isbn}</Td>
+              </Tr>
+            </Tbody>
+          </Table>
+        </VStack>
+      </Flex>
+      <VStack w={{ base: '100%', md: '10em' }} gap={{ base: '2' }} py={{ base: '5', md: '0' }}>
         <Button
           w="100%"
           colorScheme="blue"
+          color="black"
```

Comment:
```suggestion
```
Better contrast in light mode when the text is white I believe (turns black automatically in dark mode thanks to chakra)

Reply:
## White Text w/ Blue

<img width="500" alt="Capture d’écran, le 2022-05-09 à 06 41 48" src="https://user-images.githubusercontent.com/10472448/167423321-6d5732c2-3daf-4da6-bec3-4da2ebe2566b.png">
<img width="500" alt="Capture d’écran, le 2022-05-09 à 06 41 58" src="https://user-images.githubusercontent.com/10472448/167423332-3853330f-efff-42b2-b0f3-a5c284f9e87d.png">

## Black Text w/ Blue

<img width="500" alt="Capture d’écran, le 2022-05-09 à 06 42 16" src="https://user-images.githubusercontent.com/10472448/167423334-7c8bda1b-51e9-42f2-a790-b48f501b50a5.png">
<img width="500" alt="Capture d’écran, le 2022-05-09 à 06 42 24" src="https://user-images.githubusercontent.com/10472448/167423337-799bd374-039a-4bfd-8dee-96408acee8af.png">

Reply:
I wonder if an issue should be raised with Chakra if that's the case. AFAIK the `colorScheme` prop should change the text colour appropriately to match a11y needs